.github/workflows/hera_pr.yaml

```
@@ -3,16 +3,19 @@ name: Hera PR build
 on: pull_request
 
 jobs:
-  hera-pr-build:
-
+  build:
+    name: build py${{ matrix.python-version }} on ${{ matrix.os }}
     timeout-minutes: 10
-
-    runs-on: ubuntu-latest
-
     strategy:
       fail-fast: false
       matrix:
+        os: [linux, macos]
```

This line should be 
```suggestion
        os: [ubuntu-latest, macos-latest]
```
or we can specify version like `ubuntu-20.04, macos-11`

ref
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on

---

done